### PR TITLE
[UPDATE] Upgraded to Ember-cli 2.4.2 and added ember-try scenarios for 2.2, 2.3 & 2.4

### PR DESCRIPTION
**Pull request by kiwiupover**
Also bumped version to `0.3.1`

### CODE REVIEW

**Comment by kiwiupover** [comment]
I will look into the test failures

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-route-action-helper",
-  "version": "0.3.0",
+  "version": "0.3.1",
```
Comment (poteto) [inline package.json]: Don't bump this, please.

**Comment by kiwiupover** [comment]
@poteto removed the version bump

**Comment by poteto** [comment]
Thanks! @homu r+

**Pull request merged**